integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDynAsConfigCluster.java

```diff
@@ -0,0 +1,183 @@
+
+    // In OKD cluster, we need to get the routeHost for the external admin service
+    String routeHost = createRouteForOKD(getExternalServicePodName(adminServerPodName), domainNamespace);
+    
+    logger.info("Validating WebLogic admin server access by login to console");
+    testUntil(
+        assertDoesNotThrow(() -> {
+          return adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, routeHost);
+        }, "Access to admin server node port failed"),
+        logger,
+        "Console login validation");
+
+    // create a new dynamic cluster using an online WLST script
+    createNewDynCluster();
+
+    // verify the managed server pod in newly added dynamic cluster comes up
+    checkPodReadyAndServiceExists(newManagedServerPodPrefix + 1, domainUid, domainNamespace);
+  }
+
+  private void createNewDynCluster() {
+    // get admin service hostname
+    String adminSvcExtHost = createRouteForOKD(getExternalServicePodName(adminServerPodName), domainNamespace);
+    logger.info("admin svc host = {0}", adminSvcExtHost);
+
+    // get port for default channel
+    logger.info("Getting port for default channel");
+    int defaultChannelPort = assertDoesNotThrow(()
+        -> getServicePort(domainNamespace, getExternalServicePodName(adminServerPodName), "default"),
+        "Getting admin server default port failed");
+    assertNotEquals(-1, defaultChannelPort, "Couldn't get valid port for default channel");
+    logger.info("default channel port: {0}", defaultChannelPort);
+
+    // create WLST property file
+    File wlstPropertiesFile = assertDoesNotThrow(() -> File.createTempFile("wlst", "properties"),
+        "Creating WLST properties file failed");
+
+    Properties p1 = new Properties();
+    p1.setProperty("admin_username", ADMIN_USERNAME_DEFAULT);
+    p1.setProperty("admin_password", ADMIN_PASSWORD_DEFAULT);
+    p1.setProperty("admin_host", adminServerPodName);
+    p1.setProperty("admin_port", Integer.toString(defaultChannelPort));
+    p1.setProperty("new_cluster_name", newClusterName);
+    p1.setProperty("new_ms_name_prefix", newManagedServerPrefix);
+    assertDoesNotThrow(() -> p1.store(new FileOutputStream(wlstPropertiesFile), "wlst properties file"),
+        "Failed to write the WLST properties to file");
+    logger.info("WLST property file is: {0} ", wlstPropertiesFile.getAbsolutePath());
+
+    // create a new dynamic cluster using an online WLST script
+    logger.info("Creating a new dynamic cluster using an online WLST script");
+    Path createDynClusterScript = Paths.get(RESOURCE_DIR, "python-scripts", "create-dynamic-cluster.py");
+    executeWLSTScript(createDynClusterScript, wlstPropertiesFile.toPath(), domainNamespace);
+
+    // patch the domain to increase the introspectVersion value to triage the introspector and verify
+    String introspectVersion = assertDoesNotThrow(() -> getNextIntrospectVersion(domainUid, domainNamespace));
```

Comment:
Should we use the patchDomainResourceWithNewIntrospectVersion method in oracle.weblogic.kubernetes.actions.impl.Domain class?

Reply:
hmm, a good question. This method is there for a while and used by so many test suites. I will discuss it with Pani and create a new JIRA to move it to DomainUtil.java if he ok it

Reply:
and this test suite doesn't use it, patchDomainResourceWithNewIntrospectVersion :(

Reply:
JIRA https://jira.oraclecorp.com/jira/browse/OWLS-101203 is created to address this issue. Thanks